### DISCUSSION TITLE
Controller with React-Select not focused correctly

### DISCUSSION THREAD

**Discussion by nassif7** [react-hook-form/react-hook-form#9701] [General]
Hello there, 
I have a form with some fields are registered using the `register` method and some fields are registered using the `Controller`
the field which is using the `Controller` is rendering a `React-Select` component. 

the issue is when trying to validate onSubmit 
the focus never set on the `Select` even it is the first field. 
it always focus on the fields registered with `register` 

please check the code here 

https://codesandbox.io/s/determined-currying-71fy3w?file=/src/Form.jsx

**Comment by ypanagidis** [comment]
I have the same problem

**Reply by leapful** [reply]
@yiannis-pan 

Please provide a codesandbox url to re-produce the issue.

**Reply by ypanagidis** [reply]
I found my issue! I will provide screenshots later but essentially I wasn't exposing the proper ref
